src/print.jl

```diff
@@ -484,21 +484,21 @@ function function_string(print_mode, constraint::AbstractConstraint)
 end
 
 function in_set_string(print_mode, set::MOI.LessThan)
```

Comment:
Unexport?

Reply:
It is part of the extension API, when an extension defines a new set, it can a a method to this function to give a nice printing